lib/Image/Utils.pm

```diff
@@ -0,0 +1,92 @@
+package Image::Utils;
```

Comment:
It strikes me that there's significant duplication between this and https://github.com/hathitrust/feed/blob/master/lib/HTFeed/Stage/ImageRemediate.pm -- maybe worth thinking eventually how we could refactor these into a common shared dependency at some point.